sqlmesh/core/loader.py

```diff
@@ -39,6 +42,9 @@
     from sqlmesh.core.context import Context
 
 
+SQLMESH_MOCKED_COLUMN = "__SQLMESH_MOCKED_COLUMN__"
```

Comment:
maybe something like

__SQLMESH_MOCK_STAR__

Reply:
Ah so it'll generate a `*` instead of a random column name? That could be a good idea

Reply:
Ah actually I'm not sure if we can do that, because in `SqlModel`'s definition of `columns_to_types` we have:

```python
        if "*" in self._columns_to_types:
            return None
```

Reply:
no, just so that it says star

Reply:
Why would it say star?